internal/rbd/nodeserver.go

```diff
@@ -413,7 +413,23 @@ func (ns *NodeServer) stageTransaction(
 	}
 
 	if volOptions.isEncrypted() {
+		if volOptions.Mounter == rbdNbdMounter {
+			stdout, stderr, saveErr := util.ExecCommand(ctx, "sh", "-c", "ps aux | grep rbd-nbd")
+			if saveErr != nil || stderr != "" {
+				log.WarningLog(ctx, "rbd-nbd: ps command failed, error %v, ps output stdout:%q stderr: %q", saveErr, stdout, stderr)
+			} else {
+				log.UsefulLog(ctx, "rbd-nbd: ps command succeeded, stdout: %q", stdout)
+			}
+		}
 		devicePath, err = ns.processEncryptedDevice(ctx, volOptions, devicePath)
+		if volOptions.Mounter == rbdNbdMounter {
+			stdout, stderr, saveErr := util.ExecCommand(ctx, "sh", "-c", "ps aux | grep rbd-nbd")
+			if saveErr != nil || stderr != "" {
+				log.WarningLog(ctx, "rbd-nbd: ps command failed, error %v, ps output stdout:%q stderr: %q", saveErr, stdout, stderr)
+			} else {
+				log.UsefulLog(ctx, "rbd-nbd: ps command succeeded, stdout: %q", stdout)
```

Comment:
From the CI logs we see `ps` shows `rbd-nbd` as alive, but it could be possible that `rbd-nbd` is exiting after this line or may be later in time.